R/paginate.R

```diff
@@ -215,8 +212,31 @@ req_perform_iteratively <- function(req,
   vctrs::list_unchop(out)
 }
 
+#' Retrieve the next request from an iterative response
+#'
+#' In most case you should use [req_perform_iteratively()] but you can
+#' use this lower-level helper if you need more control.
```

Comment:
This seems a bit too short. Maybe something like:

In most case you should use [req_perform_iteratively()] but you can use this lower-level helper to iterate through the requests and perform them yourself with [req_perform()]